realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -642,6 +642,10 @@ protected void finalize() throws Throwable {
         super.finalize();
     }
 
+    public SharedRealm getSharedRealm() {
```

Comment:
There are two "table initialization" methods that do identical things: `initTable` and `createRealmObjectSchema`.  Since we generate these methods per-table +1, it seems worthwhile (given the method limit) to eliminate one or the other.  I went with the new one `createRealmObjectSchema`